### DISCUSSION TITLE
Let's connect 🤝

### DISCUSSION THREAD

**Discussion by maisarissi** [microsoftgraph/msgraph-sdk-dotnet#1895] [General]
Hello everyone!

I would love to connect to any of you that might be interesting in providing feedback about the Microsoft Graph .NET client library.
Please, feel free to contact me by emailing maisarissi@microsoft.com or [booking ](https://outlook.office.com/bookwithme/user/339b6f5df7f84d7f9d297ab79e924925@microsoft.com?anonymous&ep=plink)time in my calendar for us to chat.

Also, I'm thinking about creating a new email list to share any updates about Microsoft Graph .NET client library. Would you be willing to join? Let us know and share your contact in the comments (or if you don't want it public, share with me sending an email) 😄

**Comment by NoraBo** [comment]
Dear all,

I have not managed to find detailed documentation on how to use GraphServiceClient in .NET core Web API. 

The concern I currently have is what is the best approach of reusing GraphServiceClient  in .NET core Web API to make sure that the connection pool is not exhausted. 

- Is it safe to create GraphServiceClient as a singleton?  What are the  long-term effects of DNS changes. In Microsoft documentation, it is stated that we can have a single client for the lifetime of the application (https://learn.microsoft.com/en-us/graph/sdks/create-client?tabs=csharp)
- Do we have an equivalent of HttpClientFactory when instantiating GraphServiceClient? If this is the case, how can it be used and do you have any documentation for that?

Thanks
